docker/test/performance-comparison/report.py

```diff
@@ -258,20 +258,20 @@ def print_test_times():
 
 if slow_average_tests:
     #status = 'failure'
-    message_array.append(str(slow_average_tests) + ' too long')
+    message_array.append(str(slow_average_tests) + ' 🕐')
 
 if faster_queries:
-    message_array.append(str(faster_queries) + ' faster')
+    message_array.append(str(faster_queries) + ' 🐇')
```

Comment:
That's cool (but barely visible).

PS. Checked the results of performance comparison.
The difference is attributed by the changes in these tests.

Reply:
I want to have something short so that the entire result fits in the checks window. Maybe up/down arrows...

Reply:
Rabbit/Turtle is great, let's keep it.

Reply:
```
GithubException: 422 {u'documentation_url': u'https://developer.github.com/v3/repos/statuses/#create-a-status', u'message': u'Validation Failed', u'errors': [{u'field': u'description', u'message': u'description contains unicode characters above 0xffff', u'code': u'custom', u'resource': u'Status'}]}
```

Looks like we're out of luck on this :)